airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -76,12 +77,7 @@
 RESOURCE_EVENT_PREFIX = "dag_run"
 
 
-@security.requires_access(
-    [
-        (permissions.ACTION_CAN_EDIT, permissions.RESOURCE_DAG),
-        (permissions.ACTION_CAN_DELETE, permissions.RESOURCE_DAG_RUN),
-    ],
-)
+@security.requires_access_dag("DELETE", DagAccessEntity.RUN)
```

Comment:
Makes me wonder if it’s worthwhile to have a `requires_access_dag_run` shorthand

Reply:
I guess it really depends on style but the reason I did that is I like having a mapping 1:1 between the decorators `@security.requires_access_*` and APIs in auth manager

Reply:
Also all entities related to DAGs (task instances, dag runs, task logs) have a common logic so it makes sense to have one decorator for all